build.gradle

```diff
@@ -4,7 +4,7 @@ subprojects {
   apply plugin: "signing"
 
   group = "com.marklogic"
-  version = "1.3-SNAPSHOT"
+  version = "1.3.0"
```

Comment:
Can you actually change this to 1.4-SNAPSHOT? I think Jenkins gets cranky if this is not a SNAPSHOT when it tries to publish to our internal Nexus.